sdk/storage/azure-storage-blob/CHANGELOG.md

```diff
@@ -25,6 +25,8 @@ and
 - Renamed `setTier` to `setAccessTier` from `BlobAsyncClientBase` and `BlobClientBase` classes.
 - Added `ParallelTransferOptions` to buffered upload, upload from file and download to file methods.
 - Removed `Metadata` class and uses Map<String, String> for `matadata` field of `BlobProperties` and `ContainerProperties`.
+- Removed generate SAS token APIs.
+- Added `sasToken(String)` in client builder.
```

Comment:
Add a comment about removing `SASTokenCredential` and `SASTokenCredentialPolicy`